doc/md/cloud/getting-started.md

```diff
@@ -279,9 +282,9 @@ Under 'Settings' > 'Members', you can invite team members to your organization.
 These members will receive an email with a link to Atlas Cloud, and will be required to sign up with the same email
 in order to access the organization.
 
-### Regenerating Tokens
-It is possible to regenerate the access token, however once you do so the old token will be **invalidated**.
-When choosing to regenerate the token, you must remember to copy the new one into your GitHub project's 'Secrets'.
+### Deleting Tokens
+It is possible to delete the access token, however once you do so the token will be **invalidated**.
+Before you delete the token, you must remember to generate a new token, and copy the new token into your GitHub project's 'Secrets'
 
```

Comment:
```suggestion
```
No need to add this sentence. Maybe write that if you delete a bot all of its tokens will be invalidated.